upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -105,6 +106,15 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	}
 
 	dest["ProxyEnv"] = tf.ProxyEnv
+
+	if tf.cluster.Spec.Networking != nil && tf.cluster.Spec.Networking.Flannel != nil {
+		flannelBackendType := tf.cluster.Spec.Networking.Flannel.Backend
+		if flannelBackendType == "" {
+			glog.Warningf("Defaulting flannel backend to udp (not a recommended configuration)")
```

Comment:
Added a release note.  The docs we have for networking are not terribly instructive at the moment... that might have to be another PR though.